aws-lambda-scorer-python/lambda-template/Makefile

```diff
@@ -0,0 +1,25 @@
+
+.PHONY: help clean package
+
+# Location of mojo wheel
+MOJO_WHL := daimojo-*.whl
+
+help:
+	@echo
+	@echo "  help           display this help"
+	@echo "  clean          clean existing artifacts"
+	@echo "  package        create a lambda deployment package named lambda_function.zip"
+	@echo
+
+clean:
+	rm -rf aws_lambda_env
+	rm -rf pip_package
+
+package: clean
```

Comment:
I am not sure how we are going to integrate this with the main gradle build system of the repo.
Currently we build stuff on pure java jdk docker image where this would clearly not work :o(.

Reply:
Agree, we might need a different system other than jdk specific

Reply:
PyGradle could be a potential solution to integrating this project with the main gradle build system of the repo. PyGradle leverages Gradle and is an enterprise Python build system. You could look at switching Makefile with Pygradle.
https://github.com/linkedin/pygradle